scripts/download-backup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+# Usage: TMP=FOLDER scripts/download-backup.sh DELAY_IN_DAYS
+# Example: TMP=/ext/tmp scripts/download-backup.sh 1
+
+# arguments and environment variables to influence behavior
+delay="${1:-1}" # default to 1 day back
+TMP="${TMP:-/ext/tmp/}"
+
+backupLocation='s3://elife-app-backups/journal-cms/'
+selectedMonth="$(date -d "-$delay days" +%Y%m)"
```

Comment:
this must be made portable across Linux/Mac

Reply:
I amended this locally to work for me: `selectedMonth="$(date -v-$(delay)d +%Y%m)"`

The above solution would not work on Linux.